src/Scene/BoundingBox.js

```diff
@@ -138,9 +138,16 @@ BoundingBox.prototype.offsetScale = function offsetScale(bbox) {
     if (bbox.crs() != this.crs()) {
         throw new Error('unsupported offscale between 2 diff crs');
     }
-    var pitX = Math.abs(bbox.west() - this.west()) / this._dimension.x;
-    var pitY = Math.abs(bbox.north() - this.north()) / this._dimension.y;
-    var scale = bbox._dimension.x / this._dimension.x;
+
+    const dimension = {
+        x: Math.abs(this.east() - this.west()),
+        y: Math.abs(this.north() - this.south()),
+    };
+
+    var pitX = Math.abs(bbox.west() - this.west()) / dimension.x;
```

Comment:
I don't think you want to use `Math.abs` here.
Instead you should check that inputs are valid (ie : that `bbox` is contained in `this`) and throw an error otherwise